docs/multilang/configuring-credential-providers.md

```diff
@@ -8,30 +8,30 @@ This document should help multilang customers configure a suitable `CredentialPr
 
 ## Sample Provider Configuration
 
-In a Properties file, an `AWSCredentialsProperty` configuration might look like:
+In a Properties file, an `AwsCredentialsProperty` configuration might look like:
```

Comment:
Let's call out these are v2 credential providers, and maybe provide a link to the public doc that has a list of v2 providers

Reply:
Added documentation a few lines below.